web/src/app/modules/views/side/modules/links-actions/components/links-actions.component.ts

```diff
@@ -54,4 +56,24 @@ export class LinksActions {
     public get canExportToALM(): boolean {
         return this.additionalInformationService.canExportToALM;
     }
+
+    public toggleDescription() {
+        this.descriptionVisible = !this.descriptionVisible;
+    }
+
+    public get descriptionCollapsibleLabel(): string {
+        if (this.requirement.description.length > 30 && !this.descriptionVisible) {
+            return 'show more';
+        }
+
+        return '';
+    }
+
+    public get requirementDescription(): string {
+        if (this.requirement.description.length > 30 && !this.descriptionVisible) {
+            return Strings.truncate(this.requirement.description, 30);

```

Comment:
Refactor numer 30 into ´´´config.ts´´´. Maybe you can look for a similar number in this file.